container/libcontainer/handler.go

```diff
@@ -499,6 +504,7 @@ func newContainerStats(libcontainerStats *libcontainer.Stats, withPerCPU bool) *
 		setCpuStats(s, ret, withPerCPU)
 		setDiskIoStats(s, ret)
 		setMemoryStats(s, ret)
+		setPidsStats(s, ret)
```

Comment:
I didn't realize this was something we were not already collecting.  Can you add this to ignoremetrics, and ignore it by default?